core/src/components/picker-legacy/picker.tsx

```diff
@@ -202,6 +203,11 @@ export class Picker implements ComponentInterface, OverlayInterface {
   }
 
   componentDidLoad() {
+    printIonWarning(
+      'ion-picker-legacy and ion-picker-legacy-column have been deprecated in favor of the ion-picker and ion-picker-column components. These new components display inline with your page content allowing for more presentation flexibility than before.',
```

Comment:
Is this wording perhaps a bit confusing given that the developer would have just changed all their uses _from_ "ion-picker"? I've provided one suggestion.

```suggestion
      'ion-picker-legacy and ion-picker-legacy-column have been deprecated in favor of new versions of the ion-picker and ion-picker-column components. These new components display inline with your page content allowing for more presentation flexibility than before.',
```